hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/TestHoodiePositionBasedFileGroupRecordBuffer.java

```diff
@@ -125,17 +126,17 @@ public void prepareBuffer(RecordMergeMode mergeMode) throws Exception {
         Option.empty(), metaClient.getTableConfig()));
     TypedProperties props = new TypedProperties();
     props.put(HoodieCommonConfig.RECORD_MERGE_MODE.key(), mergeMode.name());
+    props.setProperty(HoodieMemoryConfig.MAX_MEMORY_FOR_MERGE.key(),String.valueOf(HoodieMemoryConfig.MAX_MEMORY_FOR_MERGE.defaultValue()));
```

Comment:
`1024 * 1024 * 1000 ` replace defaultValue

Reply:
@codope asked me to do this in his comments 
https://github.com/apache/hudi/pull/11455#discussion_r1638968211
https://github.com/apache/hudi/pull/11455#discussion_r1638968494